app/src/main/java/com/example/cr554/newsapp/NewsViewHolder.java

```diff
@@ -37,6 +39,8 @@ public void bindNews(NewsArticle news){
     }
 
     public void onClick(View view){
-        //opens a webview of the article
+        NewsArticle newsArticle = MainActivity.newsArticles.get(getAdapterPosition());
+        Intent browserIntent = new Intent("android.intent.action.VIEW", Uri.parse(newsArticle.getUrl()));
```

Comment:
For the WebView: would R.string.directory_address have to be the URL of the story being clicked on?If so is there a "best" way to edit a resource? The URL of each news-article is obtained through the JSON request for the specific article. Could i replace 
`dirWebView.loadURL(getString(R.string.directory_address))` line with:

with

'dirWebView.loadURL(newsArticle.getURL())